www/nginx/src/opnsense/scripts/nginx/ngx_functions.js

```diff
@@ -1,13 +1,15 @@
 var fs = require('fs');
 var tls_fingerprints = JSON.parse(fs.readFileSync('/usr/local/etc/nginx/tls_fingerprints.json'));
+// ignore GREASE cipher suite values when compiling a browser fingerprint (see rfc8701)
+const GREASE = ["0x0a0a", "0x1a1a", "0x2a2a", "0x3a3a", "0x4a4a", "0x5a5a", "0x6a6a", "0x7a7a", "0x8a8a", "0x9a9a", "0xaaaa", "0xbaba", "0xcaca", "0xdada", "0xeaea", "0xfafa"];
 
 function check_cipher_array(r, browser_ciphers, fingerprint_ciphers, result) {
-  if (result.status == 'Intercepted') {
+  if (result.status.indexOf('Intercepted') !== -1) {
```

Comment:
there is an includes function in javascript to do this check and I guess it is supported by njs as well:
https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/String/includes

Reply:
`includes()` workes well. thanks!